### PR TITLE
Unit Test: Get Employee & Employee Mapper

### LINKED ISSUE #35: Unit Testing for Employee Mapper
Write unit test to make sure employee mapper is working properly.

### PR DESCRIPTION
**Pull request by bwruan**

### CODE REVIEW

File: EmployeeManagementService/EmployeeManagementService.Domain/Services/EmployeeService.cs [hunk 1]
```diff
@@ -21,19 +21,7 @@ public async Task<List<Employee>> GetAllEmployees(int page, int offset)
         {
             var employees = await _employeeRepository.GetAllEmployees(page, offset);
```
Comment (jruan1580) [inline EmployeeManagementService/EmployeeManagementService.Domain/Services/EmployeeService.cs:22]: check if employees is null or empty, return an empty list.

File: EmployeeManagementService/EmployeeManagementService.Test/Mapper/EmployeeMapperTest.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            Assert.AreEqual("123-45-6789", entity.GetNotSanitizedSSN());
+            Assert.AreEqual("jdoe", entity.Username);
+            Assert.AreEqual(new byte[32], entity.Password);
+            Assert.AreEqual("Admin", entity.Role);
+            Assert.AreEqual(false, entity.IsLocked);
+            Assert.AreEqual(0, entity.FailedLoginAttempts);
+            Assert.AreEqual(true, entity.TempPasswordChanged);
+            Assert.AreEqual(false, entity.Status);
+            Assert.AreEqual(true, entity.Active);
+            Assert.AreEqual("John Doe", entity.FullName);
```
Comment (jruan1580) [inline EmployeeManagementService/EmployeeManagementService.Test/Mapper/EmployeeMapperTest.cs:45]: check if SSN was sanitize

**Review by jruan1580 — CHANGES_REQUESTED** [review]
address comments

**Comment by bwruan** [comment]
Done

**Pull request merged**